src/Main/TextUserInterface.cpp

```diff
@@ -1658,7 +1680,7 @@ namespace VeraCrypt
 			if (decryptedLayout->HasBackupHeader())
 			{
 				// Re-encrypt backup volume header
-				Core->ReEncryptVolumeHeaderWithNewSalt (newHeaderBuffer, decryptedLayout->GetHeader(), options.Password, options.Pim, options.Keyfiles);
+				Core->ReEncryptVolumeHeaderWithNewSalt (newHeaderBuffer, decryptedLayout->GetHeader(), options.Password, options.Pim, options.Keyfiles, true);
```

Comment:
Same as above: EMVOption parameter is always set to true regardless of the actual EMV option value.

Reply:
Same as above : VC in CLI mode always checks for EMV cards